src/modules/CRUD/components/Details/Read/Read.js

```diff
@@ -13,6 +13,28 @@ const NO_FEATURE = 'CRUD.details.noFeature';
 
 const emptyStringNullOrUndef = value => ['', null, undefined].includes(value);
 
+const isTableObject = value => (
+  value.reduce((list, item) => {
+    if (!list.columns.length) {
```

Comment:
I see three cases in this `reduce()` body. A simple line of comment above each case could easily explain them.